owl/EASE-OBJ.owl

```diff
@@ -2652,6 +2681,20 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#StructuralDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#StructuralDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>Structural design is the methodical investigation of the stability, strength and rigidity of structures and considers the way in which parts of a system or object are arranged. 
+
+The basic objective in structural analysis and design is to produce a structure capable of resisting all applied loads without failure during its intended life.
```

Comment:
we do not talk about analysis here, only about design

Reply:
analysis word removed.